### DISCUSSION TITLE
survival non claimed griefprotection

### DISCUSSION THREAD

**Discussion by MightyEclipseAWE** [GriefPrevention/GriefPrevention#1301] [Q&A]
How can i disable building to prevent farming in claim world? 
Players only should be able to break blocks in thier claims.

**Comment by Jikoo** [comment]
Sounds like you want [`SurvivalRequiringClaims`](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#limiting-which-worlds-griefprevention-runs-on) claim mode.

If you're doing that, you may want to consider the return ratio (`AbandonReturnRatio`) below 1.0 so that players are incentivized to make more permanent claims instead of claiming to break/build something and then immediately abandoning the claim.
